src/Operator/Constructors.jl

```diff
@@ -701,7 +701,7 @@ end
 
 """
 ```julia
-advectionoperator(basis, mesh)
+advectionoperator(basis, mesh, wind)
```

Comment:
Does `wind` need to be a keyword argument with a default so that it can be called through the gallery?